portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,174 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+}
+
+html {
+  background-color: #fff9fa;
+  position: relative;
+  font-size: 16px;
+  position: relative; 
+  margin: 0;
+  padding: 0;
+  overflow: auto;
+}
+
+body {
+  height: fit-content;
+}
+
+/* .content {
+  margin: 10rem 4rem 40px 4rem;
+} */
+
+.intro{
+  font-family: 'Poppins', sans-serif;
+  background-color: #fff9fa;
+}
+
+.intro .padding {
+  width: 100%;
+  text-align: left;
+  margin: 10rem 4rem 40px 4rem;
+  padding-right: 25%;
+  padding-left: 25%;
+  padding-top: 100px;
+  padding-bottom: 100px;
+}
+
+.aboutme {
+  font-family: 'Poppins', sans-serif;
+  background-color: #ffffff;
+}
+
+.aboutme .padding {
+  width: 100%;
+  text-align: center;
+  margin: 10rem 4rem 40px 4rem;
+  padding-right: 50px;
```

Comment:
You can assign multiple classes to an element: <div class="normal-padding normal-text">....</div>
This way you can define these separate classes in css and reuse/combine them in html:
.normal-padding {
  padding-right: 50%;
  padding-left: 50%;
  padding-top: 100px;
  padding-bottom: 100px;
}

.normal-text {
  font-family: 'Poppins', sans-serif;
  background-color: #ffffff;
  text-align: center;
}

The advantage of this approach is that you don't repeat the same thing many times (and when you need to change properties you only change them once, don't run the risk of only updating properties in half the places). Another advantage is that your CSS does not follow the structure of your HTML (aboutcontent should within row which in turn should be within aboutme) and changing structure of HTML doesn't require changes in CSS.